### ISSUE TITLE
Using i18n with a locale 'en' as translation does not work

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1242]
Hi,

just tried to use the i18n options with the following setup:

locale_dirs=['locale/']
gettext_compact=False

locale/en/LC_MESSAGES/*.mo contains all the translated po files.

But i still get errors, because sphinx does not provide a .mo file for english.

(with some extra debug prints added, it shows what happens):

```
#!python
loading translations [en]... [Errno 2] No translation file found for domain: 'sphinx'
Language 'en' not found in 'None'
[Errno 2] No translation file found for domain: 'sphinx'
Language 'en' not found in '[snip]\lib\site-packages\sphinx-1.1.3-py2.7.egg\sphinx\locale'
[Errno 2] No translation file found for domain: 'sphinx'
Language 'en' not found in '[snip]\pkgs\<proj>\doc\actions_user\en\src\../locale/'
locale not available
```

Workaround is simple:
add the missing en/LC_MESSAGES/sphinx.mo file.

Sphinx should probably add that one to the default package, so non-english documents that have 'en' as a translated languages work out of the box.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1242
- Originally reported by: Michael Schlenker
- Originally created at: 2013-08-23T14:26:28.817

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-09-16 05:26:20+00:00_

Closes issue #1242: make it clearer that "translations not available" means only for the built-in messages, and special-case English.

→ <<cset a58c1332ff4972ab53858e9cb42fb9337f521e78>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-09-16 05:27:34+00:00_

[Michael Charlton](https://bitbucket.org/Michael): this changeset clarifies the "locale not available" message; I hope this is what you meant by the errors you are getting. Otherwise please just reopen the ticket.
